src/utils/api/api.js

```diff
@@ -0,0 +1,68 @@
+import  {fetcher} from './fetcher'
+import AsyncStorage from '@react-native-async-storage/async-storage';
+export class Api {
+  constructor(host) {
+    this.host = host;
+    this.api = `${this.host}/api`;
+  }
+
+  categories = async () => {
+    return fetcher(`${this.api}/categories?populate=*`);
+  };
+  questionsSets = async () => fetcher(`${this.api}/questions-sets?populate=*`);
+  questions =  async () => fetcher(`${this.api}/questions?populate=*`);
+
+  getBookmarks = async () => {
+    try {
+      const bookmarks = await AsyncStorage.getItem('bookmarks');
+      return JSON.parse(bookmarks);
+  } catch(e) {
+      console.log('getBookmarks error', e);
+    }
+
+  }
+
+  clearStorage = async () => {
+    try {
+      await AsyncStorage.removeItem('bookmarks');
+    } catch(e) {
+      console.log('getBookmarks error', e);
+    }
+  }
+
+  setBookmark = async (newBookmark) => {
+    try {
+      const value = {
+        question: newBookmark.question,
+        rightAnswer: newBookmark.rightAnswer,
+        help: newBookmark.help,
+      };
+      const oldValue = await this.getBookmarks() || [];
```

Comment:
oldValue - same problem, give it a proper name that describes what's inside of it